crates/verifier/src/tests.rs

```diff
@@ -103,6 +111,9 @@ fn test_verify_invalid_groth16() {
 #[serial]
 #[test]
 fn test_verify_plonk() {
```

Comment:
Will this test verification with `blake3` E2E in CI? Do we run tests with all features?

Reply:
I added a test with `-F blake3`